src/zenml/zen_server/utils.py

```diff
@@ -91,13 +86,16 @@ def zen_store() -> "SqlZenStore":
     return _zen_store
 
 
-def plugin_flavor_registry() -> PluginFlavorRegistry:
+def plugin_flavor_registry() -> "PluginFlavorRegistry":
     """Get the plugin flavor registry.
 
     Returns:
         The plugin flavor registry.
     """
     global _plugin_flavor_registry
+
+    from zenml.plugins.plugin_flavor_registry import PluginFlavorRegistry
```

Comment:
What is so bad about importing this at import time? I couldn't find an explanation.

Reply:
Update: it's basically the same as the other ones. The `zenml.integrations` module ends up being imported.